week-05-reflections/dha14-stat660.md

```diff
@@ -1,12 +1,67 @@
 
 # Questions about Assigned Quiz Problems and Recipes
+[TB Chapter 12, Question 3 ] 
+*Question (dha14-stat660): Can a format name be numeric? 
```

Comment:
In Markdown, lists only render when a space is placed between the asterisk and the text, like so:

```suggestion
* Question (dha14-stat660): Can a format name be numeric? 
```